pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -170,7 +171,12 @@ func NewWebhook(wmeta workloadmeta.Component) (*Webhook, error) {
 }
 
 // GetWebhook returns the Webhook instance, creating it if it doesn't exist
-func GetWebhook(wmeta workloadmeta.Component) (*Webhook, error) {
+func GetWebhook(
+	wmeta workloadmeta.Component,
+	_ *rcclient.Client,
+	_ <-chan struct{},
+	_ string,
```

Comment:
Is there a reason why these three arguments are being passed to this function but never used ?